src/main/resources/application-dev.yml

```
@@ -99,3 +99,11 @@ server:
 vnpay:
   tmn-code: 2JVRHVF9
   hash-secret: 4TD8VD92KMWZ6E1VCUDJ4HF78LX7D1FD
+
+minio:
+  endpoint: ${MINIO_ENDPOINT:http://localhost:9000}
+  access-key: ${MINIO_ACCESS_KEY:admin}
+  secret-key: ${MINIO_SECRET_KEY:password123}
+  bucket: ${MINIO_BUCKET:vinaacademy}
+  region: ${MINIO_REGION:ap-southeast-1}
+  path-style: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)